src/components/Menu/DesktopMenu.tsx

```diff
@@ -12,6 +12,12 @@ type Props = {
 export const DesktopMenu: React.FC<Props> = ({ event, url }) => {
   return (
     <Styled.DesktopMenu>
+      <CommonStyled.MenuLink href={`/${event?.abbr}/ui`} rel="noreferrer">
```

Comment:
```suggestion
      <Link href={`/${event?.abbr}/ui`} rel="noreferrer">
```

aタグで飛んでるので、都度ページ遷移が発生してしまいます。next/linkを使うと、リンクせずにSPAの中でroutingできます。
```
import Link from 'next/link'
```

で、これを試したところ、どうもSPAの中でgetInitialPropsの評価が走ってしまうようで、 `_app.tsx` の `validateEnv()` がこけてしまう事象がでました。
こけてしまいますが、ちゃんと環境変数はbindできており、この関数を除けば問題なく動きます。
`validateEnv()` はbuild時に環境変数不足でfailさせることが目的でして、SPAとして動作している最中に実行する必要がないです。
なので、validateEnvをコメントアウトして、ここのリンクは next/link を使うのが良さそうです。

Reply:
おおお、SPAじゃないので毎回API読み込み走って微妙だなと思っていたので助かりました！！